corehq/apps/toggle_ui/views.py

```diff
@@ -180,7 +180,11 @@ def post(self, request, *args, **kwargs):
 
 def enable_vellum_beta(request):
     slug = "vellum_beta"
-    toggle = Toggle.get(slug)
+    toggle = None
```

Comment:
nit: this line is not necessary (no need to declare or initialize variables in python)